.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "library"]
+	path = library
+	url = https://github.com/h5p/h5p-php-library.git
```

Comment:
Very soon, I created it mostly just to be able to do some testing on the old version(master) and to check what is needed for the new one(2015-dev). This was before I read your code. I should probably just have tagged the old one and kept everything in master, sorry.